### PR TITLE
Finished react-redux boilerplate

### PR DESCRIPTION
**Pull request by Xhoria-RB**
# Finished the React-Redux boilerplate
It is already working but is just rendering the text in the react component

## Added:
- Created a React app  
- Axios, lodash, redux and concurrent dependencies
- Configurations to run both servers (keystone and react dev server) together
         ``` 
            npm run dev
         ```
- Created redux actions, reducers and the store
- Routes for keystone API

### CODE REVIEW

File: client/.gitignore [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
```
Comment (jesusbibieca) [inline client/.gitignore:2]: Nice! 👍

File: client/package.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "client",
```
Comment (jesusbibieca) [inline client/package.json]: What do you think about this being something like `Social Share client` or whatever but just client

File: client/src/App.test.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+
+it('renders without crashing', () => {
+  const div = document.createElement('div');
+  ReactDOM.render(<App />, div);
+  ReactDOM.unmountComponentAtNode(div);
```
Comment (jesusbibieca) [inline client/src/App.test.js:8]: `unmount`?  Why is it testing if the component is unmounted?

File: client/src/components/recipe.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+Recipe.propTypes = {
+  getRecipe: PropTypes.func.isRequired
+}
+
+const mapStateToProps = state => ({
+  recipes: state.recipes.recipes
+})
+ 
+export default connect(mapStateToProps, {getRecipe})(Recipe);
```
Comment (jesusbibieca) [inline client/src/components/recipe.js]: Missing a line at the end of file.

File: server/routes/index.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+const importRoutes = keystone.importer(__dirname);
+
+const routes = {
+  api: importRoutes('./api')
+};
+
+exports = module.exports = function(app) {
+  app.get('/api/recipe/list', keystone.middleware.api, routes.api.recipes.list);
+
+  // app.get('/api/route', (err, get) => {
```
Comment (jesusbibieca) [inline server/routes/index.js]: What about this comments?

File: server/routes/api/recipes.js [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+  Recipe.model.find((err, recipes) => {
+    res.apiResponse({
+      res: 'OK',
+      recipes
+    });
+  });
+};
+
+// rt.get('/api/route', (req, res, next) => {
+//   res.apiResponse({
```
Comment (jesusbibieca) [inline server/routes/api/recipes.js]: And this comments too?

File: client/src/App.test.js (hunk 3 above)
Comment (Xhoria-RB) [reply client/src/App.test.js:8]: That's just a default test from react, i don't know why unmount.

File: server/routes/index.js (hunk 5 above)
Comment (Xhoria-RB) [reply server/routes/index.js]: Those commets are just guidelines, i'll erase them all

File: client/package.json [hunk 7]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Social Share Client",
```
Comment (Xhoria-RB) [inline client/package.json:2]: Fixed

**Review by Xhoria-RB — COMMENTED** [review]
Update:
I delete the unnecessary comments and fixed the name in the package.json. Some files are just default files react create optimizations in the build for production, right now i don't know if they're needed, but i just leave it there

**Pull request merged**